ci-operator/step-registry/telco-e2e/telco-e2e-workflow.yaml

```diff
@@ -0,0 +1,10 @@
+workflow:
+  as: telco-e2e
+  steps:
+    pre:
+    - ref: telco-bastion-setup
+    test:
```

Comment:
I think we also want a post step that runs must-gather
and better be a pao must-gather
snapshots can be taken from 
https://quay.io/repository/openshift-kni/performance-addon-operator-must-gather?tab=tags